netcore/build.sh

```diff
@@ -36,6 +42,7 @@ skipmscorlib=false
 skipnative=false
 llvm=false
 autogen_params=''
+ci=true
```

Comment:
didn't notice this, why are we setting this to true?

Reply:
oh I see this is needed by the logging functions. we should do this like corefx does and only set it as an option in AzDO builds.

Change the .yml to pass `--ci` to build.sh